server/src/main/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequestController.java

```diff
@@ -126,12 +126,12 @@ public Mono<HttpResponse<List<FeedbackRequestResponseDTO>>> findByValues(@Nullab
                 .publishOn(Schedulers.fromExecutor(eventLoopGroup))
                 .map(feedbackReqs -> {
                     List<FeedbackRequestResponseDTO> dtoList = feedbackReqs.stream()
-                            .map(this::fromEntity).collect(Collectors.toList());
+                            .map(FeedbackRequestController::fromEntity).collect(Collectors.toList());
```

Comment:
I believe the changes to this file are superfluous. We can most likely safely abandon the changes to FeedbackRequestController and MemberProfileRepository.

Reply:
Done